ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/144_dsc_negative_no_dependant_op.robot

```diff
@@ -25,36 +25,104 @@ ${IS_NOT_PRESENT}                           1
 
 
 *** Test Cases ***
-Validate DSC and DSCI Created Without Service Mesh And Serverless Operators     #robocop:disable
+Validate DSC and DSCI Created Without Service Mesh Operator     #robocop:disable
+    [Documentation]    The purpose of this Test Case is to validate that DSC and DSCI are created
+    ...                without Service Mesh Operator installer, but with errors
+    [Tags]    Operator    Sanity    Tier1    ODS-2584
+
+    Remove DSC And DSCI Resources
+    Uninstall Service Mesh Operator CLI
+
+    Log To Console    message=Creating DSCInitialization CR via CLI
+    Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
+    Log To Console    message=Checking DSCInitialization conditions
+    Wait Until Keyword Succeeds    6 min    0 sec
+    ...    Check DSCInitialization Conditions When Service Mesh Operator Is Not Installed
```

Comment:
maybe you could rename the keyword in a way to make it clear what is the expected result.